plc4j/drivers/ads/src/main/java/org/apache/plc4x/java/ads/protocol/AdsProtocolLogic.java

```diff
@@ -1199,10 +1199,6 @@ protected CompletableFuture<PlcWriteResponse> multiWrite(PlcWriteRequest writeRe
 
     protected byte[] serializePlcValue(PlcValue plcValue, String datatypeName) throws SerializationException {
         // First check, if we have type information available.
```

Comment:
so maybe remove that comment too as it doesn't make sense anymore with the code below removed

Reply:
Good catch ... maybe a little bit of a comment to "first we try to read from the table" if nothing is found in TC2 systems the primitive types are omitted from the trable, we try if it's one of these primitive types, ... blah blah blah comments would leave some additional information for future coders.